nomad/structs/workload_id.go

```diff
@@ -52,6 +53,10 @@ var (
 	// validIdentityName is used to validate workload identity Name fields. Must
 	// be safe to use in filenames.
 	validIdentityName = regexp.MustCompile("^[a-zA-Z0-9-_]{1,128}$")
+
+	// MinNomadVersionVaultWID is the minimum version of Nomad that supports
+	// workload identities for Vault.
+	MinNomadVersionVaultWID = version.Must(version.NewVersion("1.7.0-a"))
```

Comment:
is the `1.7.0-a` literal correct? (which I guess could make sense as -a is the first possible pre-release version in semver 2)

Reply:
Ah yes, I should've copied this info from the implicit constraint:
https://github.com/hashicorp/nomad/blob/b2aa6ffd050656316d8b88b8f3ec56df8e97f907/nomad/job_endpoint_hook_implicit_identities.go#L53-L59